[xterm] Failed to build

```
In file included from ./data.h:57,
                 from ./cachedGCs.c:33:
./xterm.h:253:14: warning: conflicting types for built-in function ‘malloc’; expected ‘void *(long unsigned int)’ [-Wbuiltin-declaration-mismatch]
  253 | extern char *malloc();
      |              ^~~~~~
./xterm.h:253:14: note: ‘malloc’ is declared in header ‘<stdlib.h>’
./xterm.h:254:14: warning: conflicting types for built-in function ‘realloc’; expected ‘void *(void *, long unsigned int)’ [-Wbuiltin-declaration-mismatch]
  254 | extern char *realloc();
      |              ^~~~~~~
./xterm.h:254:14: note: ‘realloc’ is declared in header ‘<stdlib.h>’
In file included from ./xterm.h:307,
                 from ./button.c:62:
./ptyx.h:1700:9: error: unknown type name ‘uint32_t’
 1700 | typedef uint32_t MyPixel;
      |         ^~~~~~~~
In file included from ./xterm.h:307,
                 from ./data.h:57,
                 from ./cachedGCs.c:33:
./ptyx.h:1700:9: error: unknown type name ‘uint32_t’
 1700 | typedef uint32_t MyPixel;
      |         ^~~~~~~~
make: *** [Makefile:141: cachedGCs.o] Error 1
make: *** Waiting for unfinished jobs....
./button.c: In function ‘executeCommand’:
./button.c:5766:11: error: ‘EXIT_FAILURE’ undeclared (first use in this function)
 5766 |      exit(EXIT_FAILURE);
      |           ^~~~~~~~~~~~
./button.c:5766:11: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:141: button.o] Error 1
xterm failed to build: `make V=0 -j2` exited with 2
```

--- a/packages/xterm.rb
+++ b/packages/xterm.rb
@@ -3,35 +3,35 @@ require 'package'
 class Xterm < Package
   description 'The xterm program is a terminal emulator for the X Window System.'
   homepage 'https://invisible-island.net/xterm/'
-  version '361'
+  version '362'
   compatibility 'all'
-  source_url "https://invisible-mirror.net/archives/xterm/xterm-361.tgz"
-  source_sha256 '85610f20d5e47205cc1b6876f7a4da28d6ae051bd8eac0b932e92c37a73a623f'
+  source_url "https://invisible-mirror.net/archives/xterm/xterm-362.tgz"
+  source_sha256 '1d4ffe226fa8f021859bbc3007788ff63a46a31242d9bd9a7bd7ebe24e81aca2'
+
+
+
   depends_on 'pcre'
   depends_on 'sommelier'
 
   def self.build
-    ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"
-    ENV['PATH'] = '/bin:/sbin:/usr/local/bin' # The /usr/bin need to remove from PATH as bash will point 'tic' to /usr/bin/tic 
-    system "./configure",
-           "--with-x",
-           "--with-pcre",
-           "--with-xinerama",
-           "--enable-dabbrev",
-           "--enable-toolbar",
-           "--with-pkg-config",
-           "--enable-exec-xterm",
-           "--enable-16bit-chars",
-           "--enable-dec-locator",
-           "--enable-double-buffer",
-           "--enable-readline-mouse",
-           "--enable-regis-graphics",
-           "--enable-sixel-graphics",
+    system './configure',
+           '--with-x',
+           '--with-pcre',
+           '--with-xinerama',
+           '--enable-dabbrev',
+           '--enable-toolbar',
+           '--with-pkg-config',
+           '--enable-exec-xterm',
+           '--enable-16bit-chars',
+           '--enable-dec-locator',
+           '--enable-double-buffer',
+           '--enable-readline-mouse',
+           '--enable-regis-graphics',
+           '--enable-sixel-graphics',
            "--prefix=#{CREW_PREFIX}",
            "--with-xpm=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--with-ticlib"
-    system "make"
+           "--libdir=#{CREW_LIB_PREFIX}"
+    system 'make'
   end
 
   def self.install

